extensions/amp-story/1.0/amp-story.js

```diff
@@ -580,6 +581,12 @@ export class AmpStory extends AMP.BaseElement {
 
     this.initializeBookend_();
 
+    const {document} = this.win;
```

Comment:
Nit: you don't need this extra `const`, please use directly `this.win.document`

Reply:
You're right, I thought we preferred destructuring object like in #15204, but it's not the case here.